doc/man1/openssl-x509.pod.in

```diff
@@ -545,9 +545,8 @@ Clears all the permitted or trusted uses of the certificate.
 =item B<-addtrust> I<arg>
 
 Adds a trusted certificate use.
-Any object name can be used here but currently only B<clientAuth> (SSL client
-use), B<serverAuth> (SSL server use), B<emailProtection> (S/MIME email)
-and B<anyExtendedKeyUsage> are used.
+Any object name can be used here but currently only B<clientAuth>,
+B<serverAuth>, B<emailProtection>, and B<anyExtendedKeyUsage> are used.
```

Comment:
I don't understand in what sense *only* those EKUs are "used".  Whatever EKUs appear are used, and if the X509_STORE_CTX requires a particular purpose, then that purpose must match (not be rejected, or be included if an explicit inclusion list is used).  If the point being made is that the built-in purposes only employ those EKUs, that would be a more accurate statement, but OpenSSL supports custom application "purpose" definitions...

Reply:
I did not invent that formulation, but according to my (limited) understanding of the purpose vs. EKU business
I've replaced `used` by `defined` (whatever this means, but this is the wording used also in the but-next sentence).